src/app/containers/Include/index.jsx

```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react/no-danger */
+import React, { useEffect, useState } from 'react';
+import 'isomorphic-fetch';
+import { string } from 'prop-types';
+import { GridItemConstrainedMedium } from '#lib/styledGrid';
+import webLogger from '#lib/logger.web';
+import useToggle from '../Toggle/useToggle';
+
+const logger = webLogger();
+
+const IncludeContainer = ({ href }) => {
+  const [markup, setMarkup] = useState(null);
+  const { enabled } = useToggle('styIncludes');
```

Comment:
We don't currently namespace toggles by page type or pluralise feature names, so this should probably be `useToggle('include')` for consistency.